eunkyung/routes/postRouter.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const postController = require('../controllers/postController')
+
+const router = express.Router()
+
+router.post('/posting', postController.posting)
+router.get('/user/:userId', postController.userPost)
+router.get('/list', postController.postList)
+router.patch('/update', postController.updatePost)
+router.delete('/delete/:postId', postController.deletePost)
```

Comment:
이미 엔드포인트가 `(... /posts)`로 시작하기 때문에 post를 다시 명시해줄 필요가 없습니다.
추가로 지금 구현하고 있는 API는 REST 기반의 API으로 엔드포인트에 동사를 작성하지 않는 것을 권장하고 있습니다.
```suggestion
router.post('/', postController.posting)
router.get('/:userId', postController.userPost)
router.get('/', postController.postList)
router.patch('/', postController.updatePost)
router.delete('/:postId', postController.deletePost)
```